app/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,22 @@
 <resources>
     <string name="app_name">Dogbook</string>
     <string name="appName">Dogbook</string>
+    <string name="logIn">Log In</string>
+    <string name="Username">Username</string>
+    <string name="loginUsernameHint">Enter a username for your dog</string>
+    <string name="Password">Password</string>
+    <string name="loginPasswordHint">Enter your password</string>
+    <string name="newUser">New User?</string>
+    <string name="signUp">Sign Up</string>
+    <string name="confirmPassword">Confirm password</string>
+    <string name="confirmPasswordHint">Confirm your password</string>
+    <string name="ownerName">Owner\'s Name</string>
+    <string name="ownerNameHint">Enter the name of the owner</string>
+    <string name="breed">Breed</string>
+    <string name="breedHint">Enter the dog\'s breed</string>
+    <string name="birthday">Dog\'s Birthday</string>
+    <string name="looking_for_playmates">Is your dog looking for playmates?</string>
+    <string name="ownerContact">Way to contact the owner (optional)</string>
+    <string name="ownerContactHint">Email or phone</string>
+    <string name="haveAccount">Do you have an account?</string>
```

Comment:
Since this app is going to be larger, I'd consider pulling the resources out into multiple modules.  So, you'll have the resources for login, resources for the map, etc.  That'll make it easier to navigate and tell what each thing is for in the future